react/testing.md

```diff
@@ -71,27 +71,33 @@ The `data-spec` attribute is specific to testing and not tied to presentation li
 </div>
 ```
 
-Tests using `getSingleSpecWrapper` would look like:
+Tests using `getSpecWrapper` would look like:
```

Comment:
no it's always an array of nodes. it'll just be an array of 1. We simplified `getSingleSpecWrapper` & `getSpecWrappers` to just `getSpecWrapper`. Having both was confusing & `getSingleSpecWrapper` was a pain to use.